web-app/react-app/components/AuthPage/shared/Checkbox/checkbox.pcss

```diff
@@ -0,0 +1,39 @@
+.root {
+  display: flex;
+  font-size: 0.75em;
+  margin: 0;
+}
+
+.checkbox {
+  flex-shrink: 0;
+  margin-right: 0.25em;
+  width: 22px;
+  height: 19px;
+  top: 2px;
+  position: relative;
+  background: url('../../../../../images/checkbox.svg');
+  cursor: pointer;
+  appearance: none;
```

Comment:
For some reason the iPhone is not rendering the Remember me SVG properly:
<img width="369" alt="screen shot 2018-07-20 at 10 39 47" src="https://user-images.githubusercontent.com/1593398/42994186-07f64b98-8c16-11e8-9aa0-3a855a791fe8.png">

Reply:
@mondoreale

Reply:
[CORE-1499](https://streamr.atlassian.net/browse/CORE-1499)